src/base/configuration.cpp

```diff
@@ -110,8 +110,105 @@ std::string Configuration::getVersionExtra() {
   return CVC4_EXTRAVERSION;
 }
 
+std::string Configuration::copyright() {
+  std::stringstream ss;
+  ss << "Copyright (c) 2009-2017 by the authors and their institutional\n"
```

Comment:
Nit: could turn the `\n` into `std::endl`.

Reply:
This is on purpose. Makes it easier to see what the output will look like. Ideally we would have a formatter here, that splits a chunk of text into 80 chars lines.